lib/pdf_fill/forms/va686c674.rb

```diff
@@ -0,0 +1,1833 @@
+              },
+              'last' => {
+                key: 'veteranSpouse.previousSpouse.lastName[%iterator%]',
+                limit: 18,
+                question_num: 15,
+                question_suffix: 'C',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  SPOUSES PREVIOUS SPOUSE LAST NAME'
+              },
+              'suffix' => {
+                # @TODO not in the form
+              } # end of suffix
+            }, # end of full name
+            'start_date' => {
+              'month' => {
+                key: 'spouse_marriage_history.start_date.month[%iterator%]',
+                limit: 2,
+                question_num: 15,
+                question_suffix: 'A',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  PREVIOUS SPOUSE MARRIAGE DATE MONTH'
+              },
+              'day' => {
+                key: 'spouse_marriage_history.start_date.day[%iterator%]',
+                limit: 2,
+                question_num: 15,
+                question_suffix: 'B',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  PREVIOUS SPOUSE MARRIAGE DATE DAY'
+              },
+              'year' => {
+                key: 'spouse_marriage_history.start_date.year[%iterator%]',
+                limit: 4,
+                question_num: 15,
+                question_suffix: 'C',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  PREVIOUS SPOUSE MARRIAGE DATE YEAR'
+              }
+            }, # end of start date
+            'start_location' => {
+              'city' => {
+                key: 'spouse_marriage_history.start_location.city[%iterator%]',
+                limit: 18,
+                question_num: 15,
+                question_suffix: 'D',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  PREVIOUS SPOUSE MARRIAGE LOCATION CITY'
+              },
+              'state' => {
+                key: 'spouse_marriage_history.start_location.state[%iterator%]',
+                limit: 2,
+                question_num: 15,
+                question_suffix: 'E',
+                question_text: 'PREVIOUS MARRIAGE HISTORY >  PREVIOUS SPOUSE MARRIAGE LOCATION STATE'
+              }
+              # @TODO country key: 'spouse_marriage_history.start_location.country[%iterator%]'
+            }, # end of start location
+            'reason_marriage_ended' => {
+              'death' => { key: 'spouse_marriage_history.reason_marriage_ended.death[%iterator%]' },
+              'divorce' => { key: 'spouse_marriage_history.reason_marriage_ended.divorce[%iterator%]' },
+              'annulment' => { key: 'spouse_marriage_history.reason_marriage_ended.annulment[%iterator%]' },
+              'other' => { key: 'spouse_marriage_history.reason_marriage_ended.other[%iterator%]' }
+            },
+            'reason_marriage_ended_other' => {
+              key: 'spouse_marriage_history.reason_marriage_ended_other[%iterator%]',
+              # limit: 12, # @TODO THIS BREAKS
```

Comment:
Marking this one for visibility. Looks like it's not needed

Reply:
This has been removed, thanks!